CRAN submission failure

This warning is reported:

```R
Flavor: r-devel-linux-x86_64-debian-gcc
Check: whether package can be installed, Result: WARNING
  Found the following significant warnings:
    /home/hornik/lib/R/Library/4.6/x86_64-linux-gnu/RcppArmadillo/include/RcppArmadillo/version/arma.h:71:17: warning: Using fallback compilation with Armadillo 14.6.3. Please consider defining -DARMA_USE_CURRENT. See GitHub issue #475 for more. [-W#pragma-messages]
  See '/srv/hornik/tmp/CRAN_pretest/Luminescence.Rcheck/00install.out' for details.
  Used C++ compiler: 'Debian clang version 19.1.7 (3+b2)'
```

Uwe Ligges has written with a possible solution:


Comment from Dirk Eddelbuettel, the RcppArmadillo maintainer (CCed):


-----------------------------------------------------------------------------
Yes, please see the top-level issue 475 (and its sub-issues) at

   https://github.com/RcppCore/RcppArmadillo/issues/475

For your package all you should hopefully need is to add a line to
src/Makevars and src/Makevars.win (or expand one if you have it) so
PKG_CXXFLAGS also contains the term -DARMA_USE_CURRENT so that the
current Armadillo is used via RcppArmadillo.

Most packages do not need changes, a few may now get warnings about
deprecated code.  Please get in touch (via an issue at the
RcppArmadillo repo, preferably) if you need help.  You could also
choose to define -DARMA_USE_LEGACY but we want to (eventually) move
away from this version.

Let me know if you need other help. A new version 15.0.2-1 of package
RcppArmadillo is now at CRAN and should work for many packages, yours
included.  Until all CRAN binaries are rebuilt consider installing a
binary from https://rcppcore.r-universe.dev/RcppArmadillo -- and
thank you in advance for moving to (Rcpp)Armadillo 15.0.* which we
plan to make the default once all (or most) packages needing touchups
have done so.

--- a/src/Makevars
+++ b/src/Makevars
@@ -1,2 +1,2 @@
-PKG_CPPFLAGS=-I../inst/include
+PKG_CPPFLAGS=-I../inst/include -DARMA_USE_CURRENT
 CXX_STD = CXX17

--- a/src/Makevars.win
+++ b/src/Makevars.win
@@ -1,2 +1,2 @@
-PKG_CPPFLAGS=-I../inst/include
+PKG_CPPFLAGS=-I../inst/include -DARMA_USE_CURRENT
 CXX_STD = CXX17

